Automattic/crowdsignal-ui
# Move editor settings to the store

## Summary

This PR changes the way we handle the editor settings.
We're moving the settings to the store to keep it synced between the Crowdsignal editor and the Isolated Block editor.
This should solve a few styling issues, especially on previews, caused by the Isolated Block Editor overriding some injected styles.

The blocks previews and the patterns inside the block library should look better
![image](https://user-images.githubusercontent.com/7811225/189156776-984bc083-1e0d-48c4-8330-40f87a28f733.png)

![image](https://user-images.githubusercontent.com/7811225/189156951-d3ddea73-2255-4a46-aabe-6d2f307e3d85.png)

The pages previews should be more consistent with the editor's content
![image](https://user-images.githubusercontent.com/7811225/189157117-434c802b-0e34-4530-935d-0babda293b1b.png)

**Note**: On the development environment, there's still a styling issue where the theme's primary color is being overridden, making some elements, like buttons, have a black color instead of the theme color. We'll try to fix this issue on another PR.
On the production build, it should look good.

## Testing Instructions
* Checkout the branch, build the project and deploy it to your sandbox
* Make sure you're sandboxed and open/create a project using the production's URL
* Look at the styles mentioned above and check if they look good.
